CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ The following emojis are used to highlight certain changes:
 * 🛠 - BREAKING CHANGE.  Action is required if you use this functionality.
 * ✨ - Noteworthy change to be aware of.
 
+## Unreleased
+
+### Added
+
+- Added support for a debugging header `Rainbow-No-Blockcache` that is gated by the `Authorization` header and does not use the local block cache for the request
```

Comment:
Any thoughts on where the docs for this header should go?

Reply:
Maybe create `docs/HEADERS.md` ?